packages/lime-system/files/usr/lib/lua/lime/mode/apbackbone.lua

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/lua
+
+local apbackbone = {}
+
+apbackbone.wifi_mode="ap"
+
+function apbackbone.setup_radio(radio, args)
+--!	checks("table", "?table")
+
+	return wireless.createBaseWirelessIface(radio, apbackbone.wifi_mode, "name", args)
```

Comment:
here as suffix you should probably pass "backbone" or "bb" instead of "name" otherwise it will conflict with `apname` mode

Reply:
thanks && sorry for the mistake!